src/shared/containers/challenge-detail/index.jsx

```diff
@@ -213,21 +214,21 @@ class ChallengeDetailPageContainer extends React.Component {
       history.push(history.location.pathname, history.state);
     }
 
-    const recommendedTechnology = getRecommendedTags(challenge);
-    if (
-      challenge
-      && challenge.id === challengeId
-      && !loadingRecommendedChallengesUUID
-      && (
-        !recommendedChallenges[recommendedTechnology]
-        || (
-          Date.now() - recommendedChallenges[recommendedTechnology].lastUpdateOfActiveChallenges
-          > 10 * MIN
-        )
-      )
-    ) {
-      getAllRecommendedChallenges(auth.tokenV3, recommendedTechnology);
-    }
+    // const recommendedTechnology = getRecommendedTags(challenge);
```

Comment:
@sushilshinde We good to comment a lot of code block instead remove that?

Reply:
yes, for right now.